resources/js/Pages/Project/Show.vue

```diff
@@ -8,6 +8,7 @@
             ]" ></breadcrumbs>
         </template>
         <div class="card">
+          STEVE
```

Comment:
Looks like you might have accidentally committed some debugging code here.

Reply:
:facepalm:

No problem. I'll get that fixed.

Reply:
Happens to the best of us!